### PR TITLE
added dmesg output of radare2 annotations

### PR DESCRIPTION
**Pull request by sijskes**
added dmesg output of radare2 annotations

### CODE REVIEW

File: applet/src/menu.c [hunk 1]
```diff
@@ -134,23 +134,49 @@ typedef struct menu_mem_base_type menu_mem_base_t ;
 
 extern menu_mem_base_t md380_menu_mem_base[];
 
-void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int enabled) 
+void create_menu_entry_rev(int menuid, const wchar_t * label , void * green_key, void  * red_key, int e, int f ,int item_count)
```
Comment (aeickho) [inline applet/src/menu.c]: f = timeout ... see void create_menu_entry_promtg_enable_screen(void) {

  md380_create_menu_entry( md380_menu_id, wt_enable, md380_menu_entry_back+1, md380_menu_entry_back+1, 6, 2 , 1);

after 2 sec call the green function

Comment (sijskes) [inline applet/src/menu.c]: @aeickho  i ve experimented with '5' but no difference in timeout. '0' means a total hangup until poweroff with volume knob, then there is a screen refresh

Comment (aeickho) [inline applet/src/menu.c]: tnx

Comment (sijskes) [inline applet/src/menu.c]: i have several now:
    // 6,2 confirmation popup misc.
    // 6,f confirmation popup scanlist.
    // 6,f confirmation popup zone.
    // 6,1 invalid number popup.

indeed anything non-zero is automatic.

**Comment by aeickho** [comment]
@sijskes you know my branch aeickho/find_id ... there is an edit/enter  string  dialog 

... it was an concept test for call to dmr-id ... 
  my wish : create an dynamic dialog (list) with corresponding dmr-ids ... 
 my wish2: :) private call, sms, and more

**Comment by sijskes** [comment]
@aeickho interesting. i will look at it.

**Comment by aeickho** [comment]
I'll rebase it now

**Pull request merged**